tests/README.md

```diff
@@ -8,7 +8,7 @@ point the tests to the cluster.
 
 ```
 cd tests # from the git repo root folder
-go test -kubeconfig=path/to/my/config
+go test -kubeconfig=path/to/my/config -config=defaut-config.json
```

Comment:
typo `s/defaut/default`

Reply:
thanks! Fixed
(also fixed formatting for Travis...)